### PR TITLE
feat: Add custom form for speakers and sessions

### LINKED ISSUE #2166: Add custorm form for creating/editting speakers and sessions
**Describe the feature you'd like**
Add custorm form for creating/editting speakers and sessions

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anhanh11001**
Fixes #2166

### CODE REVIEW

**Comment by anhanh11001** [comment]
updated

ezgif-5-34d59946f262

**Comment by anhanh11001** [comment]
@liveHarshit please review

File: app/src/main/java/org/fossasia/openevent/general/speakercall/form/SessionIdentifier.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.fossasia.openevent.general.speakercall.form
+
+object SessionIdentifier {
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/speakercall/form/SessionIdentifier.kt]: Can it be declared as enum instead of an object?

Comment (anhanh11001) [reply app/src/main/java/org/fossasia/openevent/general/speakercall/form/SessionIdentifier.kt]: I don't know. I add the object above as string in order to match the session identifier (also string) from the server. I'm not sure how to do that with enum

Comment (anhanh11001) [reply app/src/main/java/org/fossasia/openevent/general/speakercall/form/SessionIdentifier.kt]: Screenshot already added above

**Pull request merged**